.travis.yml

```diff
@@ -1,37 +1,30 @@
 language: python
-
+dist: trusty
 # This (sudo: false) is needed to "run on container-based infrastructure" on
 # which cache: is available
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-sudo: false
-
-addons:
-  apt:
-    packages:
-     - zeroc-ice34
```

Comment:
you can't at this stage
see https://github.com/openmicroscopy/openmicroscopy/pull/5409
for more info